edge/controller/model/cluster_handlers.go

```diff
@@ -105,8 +105,8 @@ func (handler *ClusterHandler) HandleCollectEdgeRouters(id string, collector fun
 		if err != nil {
 			return err
 		}
-		association := handler.store.GetLinkCollection(persistence.FieldClusterEdgeRouters)
-		for _, edgeRouterId := range association.GetLinks(tx, id) {
+		edgeRouterIds := handler.store.GetRelatedEntitiesIdList(tx, id, persistence.FieldClusterEdgeRouters)
+		for _, edgeRouterId := range edgeRouterIds {
```

Comment:
This is the extent of the bug fix, yes? The rest of the PR is just making the ATs work again?

Reply:
Correct

Reply:
Seems pretty straightforward then, no?